packages/shared/providers.yaml

```diff
@@ -2366,6 +2366,17 @@ salesforce-sandbox:
     proxy:
         base_url: ${connectionConfig.instance_url}
     docs: https://docs.nango.dev/integrations/all/salesforce
+salesforce-experience-cloud:
+    auth_mode: OAUTH2
+    authorization_url: https://${connectionConfig.subdomain}.my.site.com/services/oauth2/authorize
```

Comment:
my.site.com? is it a real value or a placeholder?
if placeholder it should probably be `https://${connectionConfig.domain}/services

Reply:
yes, it is correct domain for `Salesforce Experience Cloud Sites`

![image](https://github.com/user-attachments/assets/ac2a9471-3780-4464-8066-91ad594ea904)

https://developer.salesforce.com/docs/atlas.en-us.chatterapi.meta/chatterapi/quickstart_communities.htm